panel/pane/plot.py

```diff
@@ -46,6 +47,12 @@ class Bokeh(PaneBase):
     Bokeh panes allow including any Bokeh model in a panel.
     """
 
+    theme = param.ClassSelector(default=None,
+                                class_=(Theme, str),
+                                allow_None=True,
+                                doc="""
```

Comment:
While not as explicit, `default=None` already forces `allow_None=True`:

```suggestion
    theme = param.ClassSelector(default=None, class_=(Theme, str), doc="""
```